src/pages/SignUp/index.jsx

```diff
@@ -17,26 +17,26 @@ const Signup = () => {
     validateForm,
     resetForm,
   } = useAuthForm('Please agree to the terms and conditions')
-  //   const { showToast } = useGlobalState()
-  //   const { isLoggedIn, dispatchUser } = useUser()
-  //   const navigate = useNavigate()
+  const { showToast } = useGlobalState()
+  const { isLoggedIn, dispatchUser } = useUser()
+  const navigate = useNavigate()
 
   const signupHandler = async (email, password) => {
     try {
-      //   const response = await signUpService(email, password)
-      //   dispatchUser({ type: actions.login, payload: response })
-      //   showToast({
-      //     message: 'Success!',
-      //     type: 'success',
-      //   })
+      const response = await signUpService(email, password)
+      dispatchUser({ type: actions.login, payload: response })
+      showToast({
+        message: 'Success!',
+        type: 'success',
+      })
       console.log(email, password)
```

Comment:
you can remove console.logs if feature is finished.

Reply:
Yes, thanks for the feedback